### ISSUE TITLE
An option for Baritone that logout when done building

### ISSUE THREAD

**Issue by ghost** [cabaletta/baritone#4259]
## Describe your suggestion
There should be a setting (you can toggle on/off) that when you done building than baritone log out of the server/world (disconnect/save and quit to title)

## Settings
Toggling logout when done building using #sel, or #build

## Context
When you want to build something big overnight and afk, I want that server is not loading all the time, So i can diconnect when done building

## Final checklist
- [ ] I have not used any OwO's or UwU's in this issue.

**Comment by LiquidDenton** [comment]
doesn't #disconnectOnArrival do that?

**Comment by ZacSharp** [comment]
No, `disconnectOnArrival` is for walking / flying to a specified coordinate (`#goto`, `#elytra` and related commands) and if it interacts with `#build` then that is a bug and probably not useful.
